app/controllers/users_controller.rb

```diff
@@ -13,11 +13,25 @@ def index
       .page(params[:page])
       .per(25)
 
-    @memberships = current_organization.members.
-      where(user_id: @users.map(&:id)).
-      includes(:account).each_with_object({}) do |mem, ob|
-        ob[mem.user_id] = mem
-      end
+    user_ids = @users.pluck(:id)
+
+    @members = current_organization.
+      members.
+      where(user_id: user_ids).
+      includes(:account, :user).
+      each_with_object({}) { |mem, ob| ob[mem.user_id] = mem }.
+      values_at(*user_ids).
+      map { |m| MemberDecorator.new(m, self.class.helpers) }
+
+    # TODO: mutate theparameters so they can be used directly on the
+    # members table.
+    #
+    # @members = current_organization.members.
+    #   joins(:account, :user).
+    #   merge(@search.result(distinct: false)).
+    #   page(params[:page]).
+    #   per(25).
+    #   map { |m| MemberDecorator.new(m, self.class.helpers) }
```

Comment:
:fire:

Reply:
👍

Reply:
Actually I left this here to remind about making the member table ransack against members itself. I will try to complete the feature, and make the whole page based on members so we don't need the additional `@users` variable here.